Modules/vis/visualisations/common/vis.helper.js

```diff
@@ -41,38 +41,7 @@ var view =
   {
     this.start = ((new Date()).getTime())-(3600000*24*time);	//Get start time
     this.end = (new Date()).getTime();	//Get end time
-  },
-  
-  'round_interval':function(interval)
```

Comment:
I only saw it being used on multigraph.js, it disrupts touch usage causing small jumps.
And is bad code to use ifs that way, should use if-else.